maxim/apis/maxim_apis.py

```
@@ -689,45 +696,205 @@ def get_folders(self) -> List[Folder]:
+                
+                mime_type = head_response.headers.get('content-type', 'application/octet-stream')
+                content_length = head_response.headers.get('content-length')
+                size = 0
+                if content_length:
+                    try:
+                        size = int(content_length)
+                    except ValueError:
+                        size = 0
+                return b"", mime_type, size
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)